plugin/src/main/java/org/wildfly/swarm/plugin/gradle/GradleArtifactResolvingHelper.java

```diff
@@ -82,7 +82,7 @@ public ArtifactSpec resolve(final ArtifactSpec spec) {
     }
 
     @Override
-    public Set<ArtifactSpec> resolveAll(final Set<ArtifactSpec> specs, boolean transitive) throws Exception {
+    public Set<ArtifactSpec> resolveAll(final Set<ArtifactSpec> specs, boolean transitive, boolean defaultExcludes) throws Exception {
```

Comment:
The gradle-implementation doesn't respect this parameter?

I ask because I faced the same problems when having swarm-libs in transitive dependencies. The swarm-lib finds it's place in the section "dependencies" in the META-INF/wildfly-swarm-manyfest.yaml and later it results in the WEB-INF/lib of the WAR created by shrinkwrap. (Yes I'm using the JAR-mode and Gradle)

This resulted in strange deployment-errors like 

> 2016-09-21 10:55:03,170 ERROR [stderr] (main) org.jboss.weld.exceptions.DeploymentException: WELD-001410: The injection point has non-proxyable dependencies: [BackedAnnotatedField] @Inject private org.wildfly.swarm.container.runtime.RuntimeDeployer.defaultDeploymentCreator